api/stream/design.go

```diff
@@ -643,8 +643,7 @@ var Source = Type("Source", func() {
 	Attribute("version", EntityVersion)
 	Attribute("deleted", DeletedEntity)
 	Attribute("disabled", DisabledEntity)
-	Attribute("sinks", Sinks)
-	Required("version", "type", "name", "description", "type", "sinks")
+	Required("version", "type", "name", "description", "type")
 })
```

Comment:
Removed `sinks` field, list of nested sinks, it it went against REST principles.